app/models/item.rb

```diff
@@ -6,4 +6,14 @@ class Item < ApplicationRecord
   belongs_to :merchant
   has_many :invoice_items
   has_many :invoices, through: :invoice_items
+
+  def self.single_finder(attribute, query)
+    if attribute == "created_at" || attribute == "updated_at"
+      Item.where("to_char(#{attribute},'yyyy-mon-dd-HH-MI-SS') ILIKE ?", "%#{query}%").first
```

Comment:
Still trying to figure out if there is a DRYer way to search my ActiveRecord database